### PR TITLE
Update information about OpenGL due to libglvnd

### PR DESCRIPTION
**Pull request by ehfd**
PR Checklist:

- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] if you are adding a new page under `docs/` or `community/`, you have added it to the sidebar in the corresponding `_sidebar.json` file
- [x] put any other relevant information below

CC @hmaarrfk @h-vetinari @isuruf 

Update the documentation so that users do not use the CDT packages and instead utilize libglvnd. Any updates in the branch are encouraged.

### CODE REVIEW

File: docs/maintainer/knowledge_base.md [hunk 1]
```diff
@@ -715,9 +710,9 @@ practice has been deprecated.) To add a new CDT, make a PR on the
    For example: a new `glibc` package means we would have to edit the elf interpreter of
    all the conda package binaries.
 
-<a id="what-s-are-some-good-examples"></a>
+<a id="what-are-some-good-examples"></a>
```
Comment (isuruf) [inline docs/maintainer/knowledge_base.md:713]: We should keep these ids stable.

Comment (ehfd) [reply docs/maintainer/knowledge_base.md]: Changed to keep them as-is.

**Comment by h-vetinari** [comment]
Thanks for the PR! I'm not an expert in the openGL space, but the changes LGTM!

**Comment by isuruf** [comment]
Thanks

**Pull request merged**